### ISSUE TITLE
iOS Move File Action

### ISSUE THREAD

**Issue by wuenschedesign** [owncloud/ios-app#220]
### Steps to reproduce
1. Start ownCloud App
2. Select any ownCloud Account
3. Select Options Menu at any File
4. Select »Move«

### Expected behaviour
The user should be able to select the folder, where he wants to move the file. He should be able to navigate the directory easily and understand which folder is currently selected. Optionally he should also be shown, which file he is currently moving.
When the file was successfully moved, it should be shown to the user.

### Actual behaviour
The screen shown when moving a file is overloaded and confuses the user

### Client
**iOS version:**
12.0.1

**ownCloud app version:**
1.0 build 85

**Device model:**
iPhone 7

### Screenshot:
Current Screen | Solution Screen 1 | Solution Screen 2
-- | -- | --
![bildschirmfoto-2019-01-14-um-18 53 27-a](https://user-images.githubusercontent.com/40062503/51200945-1df58b80-18fb-11e9-9bf6-d88f7469815b.png) | ![bildschirmfoto-2019-01-14-um-18 53 27-solution-1](https://user-images.githubusercontent.com/40062503/51206833-61ef8d00-1909-11e9-9cc1-765ab3be7279.png) | ![bildschirmfoto-2019-01-14-um-18 53 27-solution-2](https://user-images.githubusercontent.com/40062503/51206850-6e73e580-1909-11e9-952f-8e244aa65c4e.png)

**Comment by michaelstingl** [comment]
@wuenschedesign Yeah, this is more like the iOS Files app. Still has some complexity, but maybe users will learn it over time…

**Comment by jesmrec** [comment]
I think some of the are interesting to have, for example, create a folder while browsing to select it as target is usual to find in many apps. Also filtering (not searching, ftm) in case the list of folders inside a folder is very long, in order to get quickly the target folder

Also, with the suggested approach, the folder tree is indented to the right for each deep-level, therefore long folder names will not be displayed complete. For a 4 or 5 level deep in portrait mode, i guess it will not be readable.

The three dot button in the cells is totally expendable, agreed with that

**Comment by wuenschedesign** [comment]
@jesmrec the current iOS interface cuts long names and is moving the list to the left with a gentle transition when a subfolder is selected. I just tried to find a solution which is close to iOS. I add a new Screen including a search/filter option if in any case the list should be long.
![bildschirmfoto-2019-01-14-um-18 53 27-solution-3](https://user-images.githubusercontent.com/40062503/51258306-f9a4b800-19a9-11e9-9ff2-2679607a31b4.png)

**Comment by wuenschedesign** [comment]
When the user uploads a file which is already in the folder, the current ownCloud App shows a confusing notification/error warning. It should give the user the possibility to decide if he wants overwrite the existing file, upload a duplicate or even cancel the action

Current Screen | Possible Screen
-- | --
![img_8258-comments](https://user-images.githubusercontent.com/40062503/51260767-48a11c00-19af-11e9-8231-79554496ea15.png) | ![img_8255](https://user-images.githubusercontent.com/40062503/51260868-7be3ab00-19af-11e9-9c22-19f252dcbd65.PNG)

**Comment by jesmrec** [comment]
the error handling, can be also discussed in here: https://github.com/owncloud/ios-app/issues/109
